### ISSUE TITLE
About Dropout

### ISSUE THREAD

**Issue by wcy1122** [ppengtang/pcl.pytorch#39]
Thanks for your outstanding work first.
You use dropout in the 2 fc_layer in your caffe version code [here](https://github.com/ppengtang/oicr/blob/pcl-multiGPU/models/VGG16/train.prototxt) , but not here, does it work for the model?
Thanks.

**Comment by ppengtang** [comment]
Hi, sorry I forgot whether I have tried dropout for this PyTorch version code or not. I will try it and give you feedback soon. You could also try it by yourself.

**Comment by ppengtang** [comment]
Hi, have you got numbers of using dropout? Sorry I am busy these days and thus have not got chances to try this experiment.

**Comment by wcy1122** [comment]
I forget the number, but it doesn't influence a lot.

**Comment by ppengtang** [comment]
Thanks for your update!
